spec/spec_helper.rb

```diff
@@ -1,19 +1,23 @@
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
-
+require 'codeclimate-test-reporter'
+require 'dotenv'
 require 'rspec/given'
 require 'webmock/rspec'
 
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'sherlock_homes'
 
+Dotenv.load('.env.test')
+
+if ENV['CODECLIMATE_REPO_TOKEN']
+  CodeClimate::TestReporter.start
```

Comment:
Nice call.  Seems like we can probably move 
require 'codeclimate-test-reporter'
to the body of this if block too.